pyoutline/outline/config.py

```diff
@@ -22,13 +22,18 @@
 
 from future import standard_library
 standard_library.install_aliases()
+import getpass
 import os
+import tempfile
 
 from six.moves.configparser import SafeConfigParser
 
 
 __all__ = ["config"]
 
+PYOUTLINE_ROOT_DIR = os.path.dirname(os.path.dirname(__file__))
+DEFAULT_USER_DIR = '{}/opencue/outline/{}'.format(tempfile.gettempdir(), getpass.getuser())
```

Comment:
We should probably use `os.path.join` here too.

Reply:
In anticipation of python3, I'd like to move to using pathlib. I've added a new commit to reflect this. @bcipriano Let me know your thoughts.